packages/core/src/lib/user.ts

```diff
@@ -44,7 +44,7 @@ export const findUserByUsernameAndPassword = async (
   password: string
 ): Promise<User> => {
   const user = await findUserByUsername(username);
-  assertThat(user, 'user.username_not_exists');
```

Comment:
is this still in use? if not we should remove phrases as well

Reply:
Removed since it's not been used.